### PR TITLE
[4.0] com_associations remove unused JHtml::sidebar

### PR DESCRIPTION
**Pull request by travisrisner**
Remove references to unused sidebar elements in com_associations. Also fixed a syntax error with the echo statement for the table caption on the modal template.

### CODE REVIEW

File: administrator/components/com_associations/tmpl/associations/modal.php [hunk 1]
```diff
@@ -44,128 +44,125 @@
 <form action="<?php echo Route::_('index.php?option=com_associations&view=associations&layout=modal&tmpl=component&function='
 . $function . '&' . Session::getFormToken() . '=1'); ?>" method="post" name="adminForm" id="adminForm">
```
Comment (Quy) [inline administrator/components/com_associations/tmpl/associations/modal.php]:
```suggestion
```

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 90e60ee54cd35b1bd5ca075d7d179b43efc79f37<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25264">issues.joomla.org/tracker/joomla-cms/25264</a>.</sub>

**Comment by infograf768** [comment]
I have tested this item :white_check_mark: successfully on 17ce73a3431daaba5c57cdde28644349b9260565<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25264">issues.joomla.org/tracker/joomla-cms/25264</a>.</sub>

**Comment by infograf768** [comment]
RTC<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/25264">issues.joomla.org/tracker/joomla-cms/25264</a>.</sub>

**Comment by infograf768** [comment]
Thanks.

**Comment by infograf768** [comment]
While testing this, I found an error in the modal because I forgot to take into account forced items in my PR https://github.com/joomla/joomla-cms/pull/21581

Please test https://github.com/joomla/joomla-cms/pull/25278

**Pull request merged**